diff -Nru botnet-1.5.3/Makefile.am botnet-1.5.3.new/Makefile.am
--- botnet-1.5.3/Makefile.am	Thu Sep 14 04:03:53 2000
+++ botnet-1.5.3.new/Makefile.am	Thu Sep 14 04:03:08 2000
@@ -1,6 +1,6 @@
 ## Process this file with automake to produce Makefile.in
 
-SUBDIRS = makelib example
+SUBDIRS = makelib
 
 bin_SCRIPTS = botnet-config
 
diff -Nru botnet-1.5.3/Makefile.am~ botnet-1.5.3.new/Makefile.am~
--- botnet-1.5.3/Makefile.am~	Thu Jan  1 01:00:00 1970
+++ botnet-1.5.3.new/Makefile.am~	Tue Sep 12 14:17:57 2000
@@ -0,0 +1,12 @@
+## Process this file with automake to produce Makefile.in
+
+SUBDIRS = makelib example
+
+bin_SCRIPTS = botnet-config
+
+EXTRA_DIST = \
+        License.txt \
+	botnet.txt \
+	file_id.diz \
+	todo.txt
+
diff -Nru botnet-1.5.3/example/Makefile.new botnet-1.5.3.new/example/Makefile.new
--- botnet-1.5.3/example/Makefile.new	Thu Jan  1 01:00:00 1970
+++ botnet-1.5.3.new/example/Makefile.new	Thu Sep 14 04:03:08 2000
@@ -0,0 +1,14 @@
+%.o: %.c
+	$(CC) $(CFLAGS) -c -o $@ $<
+
+#all: bot serv
+all: bot
+
+bot: bot.o
+	$(CC) $(LDFLAGS) -lbotnet -o $@ $<
+
+serv: serv.o
+	$(CC) $(LDFLAGS) -lbotnet -o $@ $<
+
+clean:
+	rm -f *.o bot serv core
diff -Nru botnet-1.5.3/example/bot.c botnet-1.5.3.new/example/bot.c
--- botnet-1.5.3/example/bot.c	Thu Sep 14 04:03:53 2000
+++ botnet-1.5.3.new/example/bot.c	Thu Sep 14 04:03:08 2000
@@ -4,7 +4,7 @@
 */
 
 
-#include "../makelib/botnet.h"
+#include <botnet.h>
 #include <stdio.h>
 #include <stdlib.h>
 #ifdef __unix__
diff -Nru botnet-1.5.3/example/serv.c botnet-1.5.3.new/example/serv.c
--- botnet-1.5.3/example/serv.c	Thu Sep 14 04:03:53 2000
+++ botnet-1.5.3.new/example/serv.c	Thu Sep 14 04:04:40 2000
@@ -3,7 +3,7 @@
   aka Ze KiLleR / SkyTech
 */
 
-#include "../makelib/botnet.h"
+#include "botnet.h"
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
